devicetypes/smartthings/zwave-battery-thermostat.src/zwave-battery-thermostat.groovy

```diff
@@ -303,7 +303,7 @@ def zwaveEvent(physicalgraph.zwave.commands.thermostatmodev2.ThermostatModeSuppo
 	if(cmd.heat) { supportedModes << "heat" }
 	if(cmd.cool) { supportedModes << "cool" }
 	if(cmd.auto) { supportedModes << "auto" }
-//	if(cmd.auxiliaryemergencyHeat) { supportedModes << "emergency heat" } // device doesn't actually support this
+	if(cmd.auxiliaryemergencyHeat) { supportedModes << "emergency heat" } // device doesn't actually support this
```

Comment:
So is it just that the device flat out doesn't support it, or doesn't support us setting it to this?

Reply:
I suppose I should remove that comment.